src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  height: 80vmin;
+  width: 80vmin;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    bottom: 50%;
+    background-color: #2c8000;
+
+    transform-origin: bottom;
+    animation: rotate 60s linear infinite;
```

Comment:
The `animation` shorthand property is used correctly here, but for better readability and maintainability, consider defining the `@keyframes` animation with a more descriptive name than 'rotate'. This will make it clearer what the animation is for, especially if there are multiple animations.